### DISCUSSION TITLE
About FTUP support in UP Function Features in PFCP Assocation Setup Response

### DISCUSSION THREAD

**Discussion by s5uishida** [edgecomllc/eupf#466] [Q&A] [answered]
Hi,

Current Open5GS does not support User Plane IP Resource Information. This is because the function was removed in release 16.3.
Therefore, in order to work with Open5GS, it is necessary to support FTUP. If not, the UPF N3 and N4 addresses must be the same.
BTW, free5GC still supports User Plane IP Resource Information, so eUPF works fine with free5GC.

I confirmed that it works with Open5GS by making the eUPF N3 and N4 addresses the same.

Could you have a plan to support FTUP?

Best regards,

--Shigeru

**Comment by s5uishida** [comment]
I tried to work eUPF with Open5GS (5GC, EPC) and free5GC (5GC) by referring to the eUPF document.

https://github.com/s5uishida/install_eupf

When working with Open5GS, I made N3 and N4 the same.
If there are any errors in my settings, I would be very grateful if you could point them out.

Best regards,

--Shigeru

**Reply by pirog-spb** [reply]
Shigeru,

First of all let me admire your very detailed and clear description. Nothig to add. You've done a great job!

BR, Alex

**Comment by pirog-spb** [comment, marked as answer]
Hi Shigeru

FTUP option support is in progress. I suggest that we can release it in a few weeks. See #120 for detailes.

BR, Alex

**Reply by s5uishida** [reply]
Hi @pirog-spb 

Thank you for your reply.
And thank you very much for your excellent efforts.

I will update my log when FTUP is supported.

Best regards,

--Shigeru
